beacon-chain/rpc/validator/server.go

```diff
@@ -89,6 +89,7 @@ func (vs *Server) WaitForActivation(req *ethpb.ValidatorActivationRequest, strea
 
 	for {
 		select {
+		// Pinging every slot for activation.
 		case <-time.After(6 * time.Second):
```

Comment:
do we choose 6 because its half of the slot? if yes, can we use `params.BeaconConfig.SecondsPerSlot / 2 ` ?

Reply:
Oh whoops, I was messing around with this. For some reason changing this value to greater than half a slot breaks E2E.